### PR TITLE
Statistics module download

### PR DESCRIPTION
**Pull request by Maximilian-Seitz**
A gradle script to download node modules, to check the translator against.

### CODE REVIEW

File: javascript/js-module-statistics/build.gradle [hunk 1]
```diff
@@ -0,0 +1,50 @@
+plugins {
+    id("kotlin")
+    id("kotlinx-serialization")
```
Comment (Schahen) [inline javascript/js-module-statistics/build.gradle]: This two lines your don't need )

File: javascript/js-module-statistics/build.gradle [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    implementation(project(":compiler"))
+    implementation(project(":module-name-resolver"))
+    implementation(project(":panic"))
+    implementation(project(":translator"))
+    implementation(project(":translator-string"))
+
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime:${gradle.kotlinSerializationRuntimeVersion}")
+
+    //https://mvnrepository.com/artifact/org.graalvm.js/js
+    implementation("org.graalvm.js:js:$gradle.graaljsVersion")
```
Comment (Schahen) [inline javascript/js-module-statistics/build.gradle]: this whole dependency section is redundant

Comment (Maximilian-Seitz) [reply javascript/js-module-statistics/build.gradle]: I deleted it

**Comment by Schahen** [comment]
Cherry-picked

**Pull request closed without merging**